programmers/p와y의-개수.js

```diff
@@ -0,0 +1,16 @@
+function solution(s){
+    var p = s.match(/p/gi);
+    var y = s.match(/y/gi);
+    
+    if (p != null && y != null ){
+        if (p.length != y.length)
+            return false;
+        else
+            return true;
+    }
```

Comment:
```suggestion
        if (p.length !== y.length) return false;

       return true;
```
if문에서 return이 있을 경우 밖에서 else를 하지 않아도 조건이 충족하지 않으면
밖으로 나가게 되기 때문에 else를 사용하지 않아도 됩니다.